Nemt/nemt.cpp

```diff
@@ -9,44 +9,49 @@ int left_point_goal, right_point_goal , x_point_goal;
 int counter;
 int counter_limit;
 
-Nemt::Nemt(int cl) {
+nemt::nemt(int cl) {
     pixy.init();
     counter_limit = cl;
     counter = 0;
 }
 
-int Nemt::giveX(int x) {
-    giveV(x);
+int nemt::give_x(int x) {
+    give_v(x);
     return x_point_goal;
 }
 
-int Nemt::giveLeft(int x) {
-    giveV(x);
+int nemt::give_left(int x) {
+    give_v(x);
     return left_point_goal;
 }
 
-int Nemt::giveRight(int x) {
-    giveV(x);
+int nemt::give_right(int x) {
+    give_v(x);
     return right_point_goal;
 }
 
-void Nemt::giveV(int block_color) {
+void nemt::give_v(int block_color) {
     blocks = pixy.getBlocks();
-    if(blocks) // if we see at least one objects we can be happy 
-    {
-        for (int i = 0; i < blocks; i++) // runnig through all objects 
-        {
-            // if object i == goal_color and is bigger that captured object that it is object with bigger priority
+    // if we see at least one objects we can be happy
```

Comment:
I don't think we need these comments here.

With comments, please always assume that the reader understands the language better than you. You should explain only parts of code that can be tricky, assume something, etc.